module/VuFind/tests/unit-tests/src/VuFindTest/RecordTab/MapTest.php

```diff
@@ -0,0 +1,191 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\RecordTab;
+
+use VuFind\RecordTab\Map;
+
+/**
+ * Map Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+class MapTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * Test getting Description.
+     *
+     * @return void
+     */
+    public function testGetDescription(): void
+    {
+        $obj = new Map();
+        $expected = 'Map View';
+        $this->assertSame($expected, $obj->getDescription());
+    }
+
+    /**
+     * Test if the tab loaded via AJAX.
+     *
+     * @return void
+     */
+    public function testSupportsAjax(): void
+    {
+        $obj = new Map();
+        $this->assertFalse($obj->supportsAjax());
+    }
+
+    /**
+     * get a Map object
+     *
+     * @return Map
+     */
+    public function getMap()
+    {
+        $mapTabDisplay = true;
+        $basemapOptions = [
+            'basemap_url' => "www.foo.com",
+            'basemap_attribution' => "bar"
+        ];
+        $mapTabOptions = [
+            'displayCoords' => true,
+            'mapLabels'     => null,
+            'graticule'     => false
+        ];
+        $obj = new Map($mapTabDisplay, $basemapOptions, $mapTabOptions);
+        return $obj;
+    }
+
+    /**
+     * Test getting Graticule.
+     *
+     * @return void
+     */
+    public function testGetMapGraticule(): void
+    {
+        $obj = $this->getMap();
+        $expected = false;
+        $this->assertSame($expected, $obj->getMapGraticule());
+    }
+
+    /**
+     * Test getting basemap Configuration.
+     *
+     * @return void
+     */
+    public function testGetBasemap(): void
+    {
+        $obj = $this->getMap();
+        $expected = ["www.foo.com","bar"];
+        $this->assertSame($expected, $obj->getBasemap());
+    }
+
+    /**
+     * Test if the tab is Active.
+     *
+     * @return void
+     */
+    public function testIsActive(): void
+    {
+        $obj = $this->getMap();
+        $recordDriver = $this->getMockBuilder(\VuFind\RecordDriver\SolrDefault::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $recordDriver->expects($this->any())->method('tryMethod')
```

Comment:
I think you can change any to exactly(2) here -- might as well be as specific as possible when it's safe to do so... or you could refactor this to use a dataProvider if you prefer. (Entirely your choice!)